CMakePresets.json

```diff
@@ -36,6 +36,19 @@
         "CMAKE_INSTALL_CACHEDIR": "var/trafficserver"
       }
     },
+    {
+      "name": "release",
+      "displayName": "Release build",
+      "description": "Release build with Ninja generator",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/build-release",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
```

Comment:
versus "Debug"?

Do -D command line options override the presets?

Reply:
You can set variables in the cache once it exists:
```
cmake --preset release
cmake -B build-release -DCMAKE_BUILD_TYPE=Debug
```

But I do not think CMake supports doing this in one step.